docs/source/msp.rst

```diff
@@ -213,7 +218,8 @@ a. ``OrganizationalUnitIdentifier``: Is the OU value that the x509 certificate n
 b. ``Certificate``: Set this to the path of the CA or intermediate CA certificate under which client
    (peer, admin or orderer) identities should be validated. The field is relative to the MSP root
    folder. This field is optional. You can leave this field blank and allow the certificate to be
```

Comment:
My suggestion:
b. ``Certificate``: (Optional) Set this to the path of the CA or intermediate CA certificate under which client
   (peer, admin or orderer) identities should be validated. The specified path is relative to the MSP root
   folder.  If you do not set this field, then the identities are validated under any CA defined in the MSP configuration, which could be desirable in the future if you need to add other CA or intermediate certificates.

The word `rotate` was really confusing me. Maybe it is important to you to keep that.

Do we have a best practice recommendation for them here? Like Normally, this field can be safely ignored?

Reply:
Thanks, I addressed the comments.
I changed the wording to say it is 'typical' to not configure the Certificate.